livre/eq/Channel.cpp

```diff
@@ -380,18 +380,14 @@ struct Channel::Impl
         const DataSource& dataSource =
                 node->getTextureDataCache().getDataSource();
         const VolumeInformation& info = dataSource.getVolumeInfo();
-        Vector3f voxelSize = info.boundingBox.getSize() / info.voxels;
-        std::string unit = "m";
-        if( voxelSize.x() < 0.000001f )
-        {
+        const Vector3f resolution = info.resolution;
+
+        const float ratio = 1.0f / dataSource.getVolumeInfo().meterToDataUnitRatio;
+        std::string unit = "um";
```

Comment:
The default unit is unknown. At least it should be something different to um because that makes the if statement below unnecessary.